docker-entrypoint.sh

```
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# This script checks /servers for any new subfolders (custom mcp servers). If it finds
+# a subfolder with python requirements.txt that have not yet been installed,
+# it installs them, then finally runs mcpo (or whatever command is passed).
+# /servers dir also allows adding custom executables, etc. on the fly.
+# TO DO: Add support for offline node modules.
+
+set -e  # Exit immediately if a command exits with a non-zero status
+
+DIR="/servers"
+if [ ! -d "$DIR" ]; then
+    echo "$DIR does not exist. Creating it..."
+    mkdir -p "$DIR"
+fi
+
+if [ -d "$DIR" ] && [ -z "$(ls -A "$DIR")" ]; then
+    # Servers directory is empty
+    echo "$DIR is empty"
+fi
+
+if ! [ -f "/app/config.json" ]; then
```

it should raise an exception instead of using example.config.json